### DISCUSSION TITLE
SIGMA DESIGNS f/64 Emulation

### DISCUSSION THREAD

**Discussion by CRTRevivalDreamer** [86Box/86Box#3412] [Ideas]
**Is your feature request related to a problem? Please describe.**
There are 2 graphics cards compatible with OS/2 Warp 3.0 out of the box: the ATi mach64GX and the WEITEK Power 9100-but only the former can be emulated on 86Box.

**Describe the solution you'd like:**
In order to easily get a picture better than VGA out of the box while running OS/2 Warp 3.0 on 86Box, the SIGMA DESIGNS f/64 must be emulable!

**Describe alternatives you've considered:**
I'd use the mach64GX and run _INSTALL.EXE_ from the disk image within _ATIM64D1.EXE_ before finishing installing OS/2 Warp 3.0.

**Additional context**
[Here's the datasheet for the chip that powered my graphics card of choice!](http://www.bitsavers.org/components/weitek/Weitek_Power_9100_Graphics_Controller_Preliminary_Nov93.pdf)

**Comment by pankoza2-pl** [comment]
if you just want better resolution in OS/2 3.0 out of the box, then the [ISA16] S3 86c928 (Metheus Premier 928) is also a good option
but good point, I would like to see the Weitek Power Graphics card emulated as well
